test/parallel/test-fs-open.js

```diff
@@ -1,27 +1,24 @@
 'use strict';
 const common = require('../common');
-var assert = require('assert');
-var fs = require('fs');
+const assert = require('assert');
+const fs = require('fs');
+
+let caughtException = false;
 
-var caughtException = false;
 try {
   // should throw ENOENT, not EBADF
   // see https://github.com/joyent/node/pull/1228
   fs.openSync('/path/to/file/that/does/not/exist', 'r');
 } catch (e) {
-  assert.equal(e.code, 'ENOENT');
+  assert.strictEqual(e.code, 'ENOENT');
   caughtException = true;
 }
-assert.ok(caughtException);
+assert.strictEqual(caughtException, true);
 
-fs.open(__filename, 'r', common.mustCall(function(err, fd) {
-  if (err) {
-    throw err;
-  }
+fs.open(__filename, 'r', common.mustCall((err, fd) => {
+  assert.ifError(err);
 }));
```

Comment:
@edsadr wow, I don't know what I was thinking about. Of course you're right, sorry about that.